### PR TITLE
Ports n stuff

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
By Trilby
Fixes snowing event not having triggered weather affects anyware
Redoes how pill bottles spawn so its cleaner
Adds a water tank to janitors room as well as 2 sinks
Fixes miss-labled tanks in janitors room
By [Humonitarian](https://github.com/discordia-space/CEV-Eris/issues?q=is%3Apr+author%3AHumonitarian) && Pinkchink
Multizlevel shooting, semi-wroking state, not enabled
By [Skegal](https://github.com/discordia-space/CEV-Eris/issues?q=is%3Apr+author%3ASkegal)
New shotgun, currently admin spawn only, has no gunparts/frame needs more work
By [Mycah142](https://github.com/discordia-space/CEV-Eris/issues?q=is%3Apr+author%3AMycah142)
Allows the liquid known as biomatter, to be used for plants
Makes a new mutation for eggplants to have egg reagent inside them, also blood tomatos now have proteen
By [Penwin0](https://github.com/discordia-space/CEV-Eris/issues?q=is%3Apr+author%3APenwin0)
Removes unused code and fixes some documention
By [MLGTASTICa](https://github.com/discordia-space/CEV-Eris/issues?q=is%3Apr+author%3AMLGTASTICa)
Makes carbon pill bottles
Buffs robo speed
By Skyfire
New Antimatter ammo box sprites

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/shotgun/ks23.dm [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/obj/item/gun/projectile/shotgun/pump/ks
+	name = "\"Kerm Streak\" SG KS-23"
+	desc = "\"Kerm Streak\", designed by Car-Van for exspensive crowd control, said to be made with reused anti-aircraft barrels."
```
Comment (ThePainkiller) [inline code/modules/projectiles/guns/projectile/shotgun/ks23.dm:3]:
```suggestion
	desc = "\"Kerm Streak\", designed by Car-Van for extensive crowd control, said to be made with reused anti-aircraft barrels."
```

File: code/modules/projectiles/guns/projectile/shotgun/ks23.dm [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	icon = 'icons/obj/guns/projectile/ks23.dmi'
+	icon_state = "ks23"
+	item_state = "ks23"
+	max_shells = 4
+	w_class = ITEM_SIZE_HUGE
+	force = WEAPON_FORCE_ROBUST
+	slot_flags = SLOT_BACK
+	matter = list(MATERIAL_PLASTEEL = 20, MATERIAL_PLASTIC = 10, MATERIAL_STEEL = 15)
+	proj_step_multiplier = 1.2
+	ammo_type = /obj/item/ammo_casing/shotgun/pellet/scrap
```
Comment (ThePainkiller) [inline code/modules/projectiles/guns/projectile/shotgun/ks23.dm:13]: ...Why is it FORCED to shoot **only** old ammo? :< If anything it should have more spread and being mostly good with slugs

File: code/modules/reagents/reagent_containers/pill.dm [hunk 3]
```diff
@@ -64,21 +64,18 @@
...
-	desc = "Highly toxic."
+	desc = "A compound of chemicals toxic to most carbon lifeforms."
 	icon_state = "pill12"
 	preloaded_reagents = list("toxin" = 50)
 
-
 /obj/item/reagent_containers/pill/cyanide
 	name = "Cyanide pill"
-	desc = "Don't swallow this."
+	desc = "A pill to help people that breath to much."
```
Comment (ThePainkiller) [inline code/modules/reagents/reagent_containers/pill.dm:75]:
```suggestion
	desc = "A pill to help people that breath too much."
```

**Review by ThePainkiller — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/guns/projectile/shotgun/ks23.dm (hunk 2 above)
Comment (Trilbyspaceclone) [reply code/modules/projectiles/guns/projectile/shotgun/ks23.dm:13]: thats not what that var does, its for when a gun is spawned via admins

**Comment by Trilbyspaceclone** [comment]
> The KS-23 is really cool and drippy and all but it's sort of like a worse "Trench" from the same company (It has distinctly one less shot). Why would you use it as opposed to the "open source design"?
> 
> EDIT: I double checked, it does _slightly_ more damage, and has the same 4+1 thing as the Trench. Trench, but 250 scrip more expensive with a damage to boot.

Its a 1:1 port at this time, it needs a frame and uniquic gimmic before being added to loot pools, its same faction do to being coverted from exl and typically thats the brand that makes this types of weapon

**Pull request merged**